app/views/smart_answers/question.html.erb

```diff
@@ -13,27 +13,14 @@
     <div class="current-question" id="current-question">
 
       <div class="question" data-debug-template-path="<%= question.relative_erb_template_path %>">
-        <h2 data-test="question">
-          <%= question.title %>
-        </h2>
         <div class="question-body">
           <% if question.body.present? %>
             <article role="article">
               <%= question.body %>
             </article>
           <% end %>
 
-          <% if question.hint.present? %>
-            <p class="hint"><%= question.hint %></p>
-          <% end %>
-
           <div class="<%= question.error && 'error' %>">
-            <% if question.error %>
-              <p class="error-message" id="current-error" role="alert">
-                <%= question.error %>
-              </p>
-            <% end %>
-
             <%= render partial: "smart_answers/inputs/#{question.partial_template_name}", locals: { question: question } %>
 
             <%= question.post_body %>
```

Comment:
There's a line near the end of this file that renders a green button - this needs replacing with the button component, I think. It renders like this on https://smart-answers-preview-pr-4132.herokuapp.com/vat-payment-deadlines/y/2020-05-31 : 

<img width="194" alt="Screen Shot 2019-09-27 at 14 50 01" src="https://user-images.githubusercontent.com/29889908/65774425-1f3bb100-e136-11e9-9eec-9d020415d285.png">

Reply:
Fixed, good spot, thanks!